### PR TITLE
feat: php language segment

### PR DESCRIPTION
**Pull request by lnu**
### Prerequisites

- [x] I have read and understood the `CONTRIBUTING` guide
- [x] The commit message follows the [conventional commits][cc] guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been added / updated (for bug fixes/features)

### Description

php language segment

[cc]: https://www.conventionalcommits.org/en/v1.0.0/#summary

### CODE REVIEW

**Review by JanDeDobbeleer — APPROVED** [review]

File: src/segment_php.go [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+func (n *php) string() string {
+	return n.language.string()
+}
+
+func (n *php) init(props *properties, env environmentInfo) {
+	n.language = &language{
+		env:        env,
+		props:      props,
+		extensions: []string{"*.php"},
```
Comment (JanDeDobbeleer) [inline src/segment_php.go]: I would also add `"composer.json", ".php-version"`

Comment (lnu) [reply src/segment_php.go]: no idea how PHP works these days :)

Comment (JanDeDobbeleer) [reply src/segment_php.go]: and I like to keep it that way 😓

File: docs/docs/segment-php.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+## Properties
+
+- home_enabled: `boolean` - display the segment in the HOME folder or not - defaults to `false`
+- display_version: `boolean` - display the php version - defaults to `true`
+- display_error: `boolean` - show the error context when failing to retrieve the version information - defaults to `true`
+- missing_command_text: `string` - text to display when the command is missing - defaults to empty
+- display_mode: `string` - determines when the segment is displayed
+  - `always`: the segment is always displayed
+  - `files`: the segment is only displayed when `*.php` files are present (default)
```
Comment (HarasimowiczKamil) [inline docs/docs/segment-php.md:35]: It is worth supplementing the documentation with file `composer.json, .php-version` etc.

Comment (lnu) [reply docs/docs/segment-php.md:35]: yes, compose and .php-version were added at last

**Pull request merged**